pkgs/development/interpreters/luajit/default.nix

```diff
@@ -62,28 +62,33 @@ let
     else buildPackages.stdenv;
 
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "luajit";
   inherit version src;
 
   luaversion = "5.1";
 
+  LUA_PATH_PATTERN = [
+    "./?.lua"
+    "./share/lua/luajit-${finalAttrs.version}/?.lua"
+    "./share/lua/luajit-${finalAttrs.version}/?/init.lua"
+  ];
+
+  LUA_CPATH_PATTERN = [
+    "./?.so"
+    "./lib/lua/${finalAttrs.luaversion}/?.so"
+    "./lib/lua/${finalAttrs.luaversion}/loadall.so"
+  ];
+
+  # to compare with the default value, run `luajit -e "print(package.path)"`
```

Comment:
this only works if you have luajit installed through something that isn't nixpkgs

Reply:
I removed it